### DISCUSSION TITLE
[New Device Support]: Onn Android Box

### DISCUSSION THREAD

**Discussion by michaelrobgrove** [PRProd/HA-Firemote#260] [New Device Family Support Request]
### Device Name

ONN Android Box

### Which compatibility mode event number works BEST with your device?

None

### Additional Comments (optional)

I am happy to run adb commands and do some tests in a few days. I have 2 version of the ONN Android 4k Boxes. The Chromecast remote works for most functions except power, input.

**Comment by PRProd** [comment]
Greetings @michaelrobgrove!

Could you send me a product link to the device you are using?

**Comment by PRProd** [comment]
@michaelrobgrove 

While I'm waiting to hear from you, I'm going to convert this issue thread to a **Discussion**.  For new device family support (which is what this appears to be) I also recommend that you make your voice heard over at Discussion #41. 

The discussion and poll over there at #41 has been used to gather community feedback, and focus on adding new device types where they are wanted the most.  In your case, I would recommend voting "other" and then writing "Onn Android Box" in a comment.

**Comment by PRProd** [comment]
Good news @michaelrobgrove! :tada: 

On Saturday [I announced that **onn.** has won the voting process](https://github.com/PRProd/HA-Firemote/discussions/41#discussioncomment-8730872) and I've begun working on adding it to the Firemote project!  If you are available, I'd love to have you as a beta tester when the time comes!

Are you interested?
